instance/templates/instance/ansible/vars.yml

```diff
@@ -0,0 +1,62 @@
+# edxapp
+EDXAPP_PLATFORM_NAME: "{{ instance.name }}"
+EDXAPP_LMS_NGINX_PORT: 80
+EDXAPP_LMS_PREVIEW_NGINX_PORT: 80
+EDXAPP_CMS_NGINX_PORT: 80
+EDXAPP_SITE_NAME: '{{ instance.domain }}'
+EDXAPP_LMS_SITE_NAME: '{{ instance.domain }}'
+EDXAPP_LMS_BASE: '{{ instance.domain }}'
+EDXAPP_PREVIEW_LMS_BASE: '{{ instance.domain }}'
+
+EDXAPP_CMS_SITE_NAME: '{{ instance.studio_domain }}'
+EDXAPP_CMS_BASE: '{{ instance.studio_domain }}'
+
+# Forum environment settings
+FORUM_RACK_ENV: 'production'
+FORUM_SINATRA_ENV: 'production'
+
+# Emails
+EDXAPP_CONTACT_EMAIL: '{{ instance.email }}'
+EDXAPP_TECH_SUPPORT_EMAIL: '{{ instance.email }}'
+EDXAPP_BUGS_EMAIL: '{{ instance.email }}'
+EDXAPP_FEEDBACK_SUBMISSION_EMAIL: '{{ instance.email }}'
+EDXAPP_DEFAULT_FROM_EMAIL: '{{ instance.email }}'
+EDXAPP_DEFAULT_FEEDBACK_EMAIL: '{{ instance.email }}'
+EDXAPP_SERVER_EMAIL: '{{ instance.email }}'
+EDXAPP_BULK_EMAIL_DEFAULT_FROM_EMAIL: '{{ instance.email }}'
+
+# Misc
+EDXAPP_TIME_ZONE: 'UTC'
+
+# Security updates
+COMMON_SECURITY_UPDATES: true
+SECURITY_UNATTENDED_UPGRADES: true
+SECURITY_UPDATE_ALL_PACKAGES: false
+SECURITY_UPGRADE_ON_ANSIBLE: true
+
+# Pin down dependencies to specific (known to be compatible) commits.
+edx_platform_repo: '{{ instance.repository_url }}'
+edx_platform_version: '{{ instance.commit_id }}'
+edx_ansible_source_repo: 'https://github.com/edx/configuration.git'
+configuration_version: 'master'
+
```

Comment:
From @bradenmacdonald : 

> Is it possible to specify a list of installed XBlocks via ansible configuration?

Yes, we can define a list of XBlocks to load by default, or any additional requirements for that matter, thanks to a PR from Matjaz actually : ) See https://github.com/edx/configuration/pull/1035

We would need to decide on a list of XBlocks to load, and see how/if we pin down their version (we could also get their latest master for now, but that would prevent having reproducible builds, if code pulled changes depending on the build time).

Alternatively, note that there is the possibility to add additional ansible configuration from the PR description - I had forgotten to mention this in the Sandbox HOWTO, fixed: https://github.com/open-craft/documentation/blob/master/sandboxes_howto.md#additional-ansible-configuration . I haven't tested this much yet though, and not at all with the loading of additional requirements or XBlocks, so let me know how that goes when you get a chance to test it.